interface-definitions/system_watchdog.xml.in

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="system">
+    <children>
+      <node name="watchdog" owner="${vyos_conf_scripts_dir}/system_watchdog.py">
+        <properties>
+          <help>Hardware watchdog configuration</help>
+          <priority>9999</priority>
+        </properties>
+        <children>
+          <leafNode name="module">
+            <properties>
+              <help>Kernel module to load for watchdog device (optional)</help>
+              <valueHelp>
+                <format>txt</format>
+                <description>Module name (e.g. 'softdog', 'iTCO_wdt', 'sp5100_tco')</description>
+              </valueHelp>
+              <constraint>
+                <regex>[a-zA-Z0-9_\-]+</regex>
```

Comment:
I don't think we should have a catch-all regex without also any `verify()` stage.

We should either add an autocomplete and constrain to known modules, or have something in `verify()` stage to check the module exists.

Perhaps code to check values against modules folder? `/lib/modules/6.x.xxx-vyos/kernel/drivers/watchdog`

Reply:
I've now added verify logic what uses `modprobe` dry-run to check that a module exists and can be loaded.

I've also added a test to check that the test logic works as expected.